bundles.json

```diff
@@ -18,13 +18,13 @@
     "platform/features/scrolling",
     "platform/features/events",
     "platform/forms",
+    "platform/persistence/elastic",
     "platform/persistence/queue",
     "platform/policy",
     "platform/entanglement",
     "platform/search",
 
     "example/imagery",
-    "example/persistence",
     "example/eventGenerator",
```

Comment:
aah, nope, but it ****should**** be really easy to switch from http-server to the above; just `npm install` and then run the above command.